sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/ResourceEvaluationStage.java

```diff
@@ -198,7 +217,7 @@ protected Resource getFulfilledResource(MesosResource mesosResource) {
         }
     }
 
-    private boolean expectedValueChanged(MesosResource mesosResource) {
-        return !ValueUtils.equal(getResourceRequirement().getValue(), mesosResource.getValue());
+    private boolean expectedValueChanged(Value requestedResource, Value existingResource) {
```

Comment:
I'm not convinced this function needs to exist.  Its kind of helping by changing the name of the function it calls.  This could be equally well accomplished by a comment.